sweep/store.go

```diff
@@ -0,0 +1,49 @@
+package sweep
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/channeldb"
+)
+
+// SweeperStore stores published txes.
+type SweeperStore interface {
```

Comment:
Do we never remove anything from the store? A small-ish amount of data, so don't see any issue with coming up with a garbage collection scheme in the future.

Reply:
Ok, leaving garbage collection out of this PR's scope

Reply:
Agreed.